core/domain/wipeout_service.py

```diff
@@ -190,10 +248,14 @@ def verify_user_deleted(pending_deletion_request):
         bool. True if all the models were correctly deleted, False otherwise.
     """
     user_id = pending_deletion_request.user_id
-    return all((
-        _verify_basic_models_deleted(user_id),
-        _verify_activity_models_deleted(user_id),
-    ))
+    role = pending_deletion_request.role
+    if role == feconf.ROLE_ID_LEARNER:
+        return _verify_profile_related_models_are_deleted(user_id)
+    else:
+        return all((
+            _verify_basic_models_deleted(user_id),
+            _verify_activity_models_deleted(user_id),
+        ))
```

Comment:
No, verify everything even for profile user, so that we are 100 % sure that everything works correctly.

Reply:
Done now.